parser_library/src/context/hlasm_context.cpp

```diff
@@ -597,6 +597,25 @@ location hlasm_context::current_statement_location() const
     }
 }
 
+utils::resource::resource_location hlasm_context::current_statement_resource_location(bool consider_macros) const
```

Comment:
isn't this basically current_statement_location?

Reply:
Alright, adjusted `current_statement_location` and using that instead.